components/automate-load-balancer/habitat/plan.sh

```diff
@@ -54,6 +54,7 @@ pkg_binds_optional=(
 )
 
 do_download() {
+  cp -f cacert.pem /hab/pkgs/core/cacerts/2020.01.01/20200306005234/ssl/certs/cacert.pem
```

Comment:
Are we sure that the destination path will available always?

Reply:
I am assuming it will be there. Need @mwrock guidance.

Reply:
Yes until automate upgrades to the refreshed packages, this path will exist.